App/Controllers/SummonersController.php

```diff
@@ -41,10 +41,13 @@ public function store(ServerRequest $request)
 
         $divisionId = intval($division->first()->getAttributes()['id']);
 
-        // @todo replace dummy data with adequate data
-        $dummyData = $leagueOfLegendsService->matchlist($summonerData['summoner_id']);
+        // @todo replace lane with actual aggregated data
+        $lane = $leagueOfLegendsService->matchlist($summonerData['summoner_id']);
 
-        $result = array_merge($dummyData, $summonerData, ['division_ranks_id' => $divisionId]);
+        $result = array_merge(
+            ['main_role_played' => $lane->raw()['matches'][0]['lane']],
+            $summonerData,
+            ['division_ranks_id' => $divisionId]);
         $result['users_id'] = $_SESSION['user']['id'];
 
         $summoner->fill($result);
```

Comment:
The matches fill logic can probably just go after the summoner fill, this way we have FK reference to the summoner as well.